machine-learning-box/NBA_metrics/nba_activity_percentile_metrics.dig

```diff
@@ -0,0 +1,85 @@
+# error notification email
+_error:
+  mail>: body.txt
+  subject: this workflow failed
+  to: [dilyan.kovachev@treasure-data.com]
```

Comment:
Could you remove your email?

Reply:
my e-mail should not be here in the final version of workflow. I think something strange is happening where you are not getting the latest version of my files from my "add_dilyan_kovachev" branch, but I am not sure how to fix this. Any suggestions?